test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -205,8 +207,9 @@ void doBlockchainTestNoLog(json_spirit::mValue& _v, bool _fillin)
 				tests[newtestname] = jObj;
 			}
 
-			//delete source test from the json
-			_v.get_obj().erase(_v.get_obj().find(testname));
```

Comment:
looks like so.  My test filling does not terminate with `it = erase(it)` way somehow.